ui/packages/docs/src/components/Frames/character_data.json

```diff
@@ -1110,5 +1110,19 @@
       "vid": "https://youtu.be/2VGB-csNGnI",
       "count": "https://docs.google.com/spreadsheets/d/1IQWPPH4rVSe9fvy2hpvufS1xi7Ni_jfZ7SL4aSudPuM/edit?usp=sharing"
     }
+  ],
+  "travelerhydro": [
+    {
+      "vid_credit": "Kolibri#7675",
+      "count_credit": "Kolibri#7675",
+      "vid": "https://youtu.be/MOyhu5YxeWA",
+      "count": "https://docs.google.com/spreadsheets/d/1_a0KTCdTkvITCoHvLzOzFbcRm1wMtDaHt7zFU9h2aPQ/edit?usp=sharing"
+    },
+    {
+      "vid_credit": "Kolibri#7675",
+      "count_credit": "Kolibri#7675",
+      "vid": "https://youtu.be/3_RxI2hK1l4",
+      "count": "https://docs.google.com/spreadsheets/d/1bvtaF1vt95lRQHxvnOL5jLbKMR683s_eHxx-q91q0Nk/edit?usp=sharing"
+    }
```

Comment:
Missing https://youtu.be/tERohQyDgmQ below this (copy everything and change vid).